gemfire-core/src/main/java/com/gemstone/gemfire/cache/client/internal/AbstractOp.java

```diff
@@ -47,7 +47,7 @@
   private final Message msg;
 
   protected AbstractOp(LogWriterI18n lw, int msgType, int msgParts) {
-    this.msg = new Message(msgParts, Version.CURRENT);
+    this.msg = new Message(msgParts, Version.CURRENT_GFE);
```

Comment:
We were compatible for at least 7.1 as part of initial rolling upgrade support effort. Perhaps our CURRENT_GFE should point to the same. As of now if 8.1 works well enough for our scenarios will stick to that since its not clear if the newer GemFire versions will continue supporting 7.1 version (or will be tested with those) since those are now unsupported versions. We may need to added 8.1 updated classes to client if there is trouble.